omnibot/routes/api.py

```diff
@@ -200,7 +200,10 @@ def slack_slash_command():
         )
         logger.error(
             msg,
-            extra={'team': team.team_id},
+            extra=merge_logging_context(
+                {'expected_team_id': team.team_id},
```

Comment:
same here for this [keyword](https://github.com/lyft/omnibot-private/blob/master/manifest.yaml#L100).